docs/index.md

```diff
@@ -45,6 +45,58 @@ where
 
 ## Get started
 
+### Naming Convention
```

Comment:
```suggestion
## Naming Convention
```

I'd also suggest moving this to below `## Custom Fields`, as it's pretty advanced and most users won't use it IMO.

Reply:
Updated